packages/paypal-commerce-integration/src/paypal-commerce-accelerated-checkout/paypal-commerce-accelerated-checkout-customer-strategy.spec.ts

```diff
@@ -196,12 +196,65 @@ describe('PayPalCommerceAcceleratedCheckoutCustomerStrategy', () => {
             }
         });
 
-        it('loads paypal accelerated checkout payment method', async () => {
+        it('loads paypal commerce accelerated checkout payment method', async () => {
```

Comment:
```suggestion
        it('loads paypalcommerce accelerated checkout payment method', async () => {
```

Reply:
I think its ok to keep it like this, otherwise I need to change paypal commerce accelerated checkout to paypalcommerceacceleratedcheckout, which is not better for reading purposes